plugins/dead_code_elimination/dead_code_elimination_main.ml

```diff
@@ -1,10 +1,11 @@
 open Core_kernel.Std
 open Bap.Std
+open Regular.Std
 open Format
 include Self()
 
 let union ~init ~f =
-  Seq.fold ~init ~f:(fun acc x ->
+  List.fold ~init ~f:(fun acc x ->
```

Comment:
FYI, there is `Set.union_list` (we needed this `union` helper just because there is no `union_sequence` function in core.